docs/tutorials/elemental_tutorial.md

```diff
@@ -115,9 +109,8 @@ oc project another-project
 ## Pods and Command Line Interface
 
 Pods are objects that run one or more containers. The containers in the pod
-share the same IP address and are guaranteed to run on one physical node so
-that they can find each other at `localhost` and can communicate through shared
-memory.
+share an IP address and they can communicate through `localhost` or shared memory.
+Consequently, they are executed in single physical node.
```

Comment:
```suggestion
Consequently, they are executed in a single physical node.
```

"Consequently" would mean "as a result of" something. I suppose the fact that a pod runs on a single node is not a consequence of it having a shared IP address and containers being able to access shared memory? It's the other way around, right?

Reply:
Correct. Changed this so that the result is a need for a single physical node.